tests/end2endtest/es_server.yml

```diff
@@ -0,0 +1,87 @@
+# Easy-server server file for pywbem end2end tests
+#
+# This file has the following format:
+#
+# vault_file: VAULT_FILE           # Path name of easy-server vault file.
+#                                  # Mandatory.
+#
+# default: NICKNAME                # Nickname of default group or server.
+#                                  # Mandatory.
+#
+# server_groups:                   # WBEM server groups.
+#                                  # Optional, default: no groups defined.
+#
+#   NICKNAME42:                    # Nickname of WBEM server group.
```

Comment:
Not really a big issue but why do we call it nickname rather than name???

Reply:
That would be a change in easy-server, since it is part of the stuff inherited from there. Also, I wanted to have a term that is sufficiently far away from the "hostname" of the server and makes it clear that it only exists in the realm of easy-server.